### ISSUE TITLE
[2021-01-07] Explain why initialCount behaves as it currently does

### ISSUE THREAD

**Issue by benjie** [graphql/graphql-wg#587]
Suggest you watch the YouTube video for full context on this; a brief summary is available in the notes (see the question from Ivan under "Item 1" in [the bulleted list](https://github.com/graphql/graphql-wg/blob/master/notes/2021-01-07.md#deferstream-updates-10m-rob)) but I think it lacks some of the nuance.

In summary: Ivan asked whether `initialCount` was a minimum or a maximum (or a suggestion), and there was discussion and arguments for both sides, so we were hoping that @Keweiqu could detail the current choice and explain the decisions that led us here. This comes out of [my request](https://github.com/graphql/graphql-spec/pull/742#discussion_r552480121) that `initialCount` be made optional so that you can `@stream` without requiring arguments.

- assignee(s): @Keweiqu 
- source: https://github.com/graphql/graphql-wg/blob/master/notes/2021-01-07.md#deferstream-updates-10m-rob

---

_Note: Action Item issues are reviewed and closed during Working Group
meetings._

**Comment by benjie** [comment]
@Keweiqu Please could you comment on this issue [so that I may assign it to you](https://github.blog/changelog/2019-06-25-assign-issues-to-issue-commenters/)? A simple ":wave:" will do :+1:

**Comment by robrichard** [comment]
@benjie I went through my notes from past meetings and we landed on specifying that servers SHOULD (not must) honor the `initialCount` provided, same behavior as previously discussed about `@defer`. I believe that's where we ended in the WG meeting as well.

We actually had already outlined this in the RFC: https://github.com/graphql/graphql-spec/blob/master/rfcs/DeferStream.md#server-requirements-for-defer-and-stream

I think I caused some confusion in the WG meeting by not explaining this clearly.
